cmd/uplink/cmd/setup.go

```diff
@@ -247,3 +277,35 @@ func ApplyDefaultHostAndPortToAddr(address, defaultAddress string) (string, erro
 	// address is host:
 	return net.JoinHostPort(addressParts[0], defaultPort), nil
 }
+
+// saveEncryptionKey saves the encryption key into a new file created in
+// filepath.
+func saveEncryptionKey(key *storj.Key, filepath string) error {
```

Comment:
Can we have this function save the human-readable encryption key instead of `storj.Key`?

Reply:
Should do we remove the check if key is zero, which with this change will be if the human-readable key is empty?

Reply:
I think you can leave it.